dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/ColumnOrganizedStachExtension.cs

```diff
@@ -94,13 +95,26 @@ private static Models.Table GenerateTable(Stach.V2.Package package, string prima
             {
                 var metadataItems = primaryTable.Data.Metadata.Items;
                 var tableMetadataLocations = primaryTable.Data.Metadata.Locations.Table;
-
                 foreach (var location in tableMetadataLocations)
                 {
                     metadataItems.TryGetValue(location, out var metadataItem);
                     if (metadataItem != null)
                     {
                         table.Metadata.Add(location, StachUtilities.ValueToString(metadataItem.Value));
+                        //table.RawMetadata.Add(location, metadataItem.Value);
+
+                        // parsing metadataItem.Value into a List of values
+                        string valString = metadataItem.Value.ToString();
```

Comment:
Isnt it possible to iterate over the Values (i guess that will be ListValue type) rather than doing a ToString() and parsing it ?

Reply:
I tried that, but when I tried to do anything with ListValue I'd get this error:
![image](https://user-images.githubusercontent.com/88855669/131691846-2363bff5-bbed-4b3a-bc8a-c47f69229731.png)
Couldn't figure out how to get the individual values without parsing it as a string